main.tf

```diff
@@ -1,3 +1,18 @@
+locals {
+  ecs_cluster_enabled = module.this.enabled && var.create_ecs_cluster ? 1 : 0
+  ecs_user_data       = <<-EOF
+    ${coalesce(var.user_data_base64, "#!/bin/bash")}
+    echo \"ECS_CLUSTER=${module.this.id}\" >> /etc/ecs/ecs.config
+  EOF
```

Comment:
One possible problem is that the default interpreter for user_data is python

So if people are passing in `user_data_base64` and it is not meant to be interpreted by bash, then this may break their userdata.

Seeking input on how to best handle this